dev/source/docs/using-sitl-for-ardupilot-testing.rst

```diff
@@ -431,9 +431,19 @@ To use a real serial device you can use a command like this:
 
 what that does it pass the --uartB argument to the ardupilot code,
 telling it to use /dev/ttyUSB0 instead of the normal internal simulated
-GPS for the 2nd UART.
+GPS for the 2nd UART. You can find the serial port mappings :ref:`here <common-pixhawk-serial-names>`
```

Comment:
Probably should be more generalized for other bds...serial->uart is usually in the boards description in common_autopilots

Reply:
I don't think this is right? The link I'm looking for is this one https://ardupilot.org/copter/docs/common-pixhawk-serial-names.html and the PX4_HAL column. They are synthetic values. I think the page is misnamed

Reply:
this is not relevant for SITL

Reply:
How about we add the uartA names etc to common-serial-options ?